apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestSchemaManager.java

```diff
@@ -148,7 +131,7 @@ public static Map<String, String> createScriptPlaceholdersMap(DataSource dataSou
         placeholders.put("logic.psql-only", "");
         placeholders.put("logic.hsql-only", "-- ");
```

Comment:
This lil fella (and its non-test counterpart) must remain until the flyway rebaselining removes the hsql specific flags, or else we blow up on migration.